routes/api/index.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const router = express.Router();
+const commentsApiRouter = require('../routes/api/comments');
+const followsAPiRouter = require('../routes/api/follows');
+const stories = require('../routes/api/stories');
+const users = require('../routes/api');
+
+
+//const app = express();
+
+//app.use('/', )
+
+
+
+
```

Comment:
A router should be created here and exported. The router should `use` all of the imported routers from the routers within `/routes/api`.

Reply:
my bad i forgot to git push before i pr'ed